CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 
+v1.76.1
+------------------------------
+*November 27, 2019*
+
+### Changed
+- `yarn.lock` file.
```

Comment:
What was the reason for this change?

Reply:
@DamianMullins - Ben tested on his machine and it gave an error. I also updated the package in menuweb and the palette changes didn't come through